docs/usage/guide_f5.rst

```diff
@@ -0,0 +1,316 @@
+:orphan: true
+
+F5 BIG-IP Guide
+===============
+
+BIG-IP systems can inspect, secure, balance, and accelerate the traffic that passes through your network.
+
+F5 provides modules for Ansible that you can use to deploy, provision, configure, and orchestrate BIG-IP systems. BIG-IP can be physical hardware or a virtual edition (BIG-IP VE) running in a public or private cloud.
+
+About the API
+`````````````
+
+The primary API that's included with BIG-IP is called iControl REST. The Python library that interacts with iControl REST is called ``f5-sdk``.
+
+To work with the F5 Modules for Ansible, you should install ``f5-sdk``, as well as:
+
+- ``bigsuds``, the F5 SOAP API that was used for modules prior to Ansible 2.2
+- ``netaddr``, which helps with address manipulation
+- ``deepdiff``, to compare and find the differences between objects
+
+
+For details:
+
+- `Learn more about the f5-sdk Python library <http://f5-sdk.readthedocs.io/en/latest/userguide/index.html>`_
+
+- `Access the f5-sdk Python library <https://github.com/F5Networks/f5-common-python>`_
+
+- `Learn more about iControl REST <https://devcentral.f5.com/Wiki/Default.aspx?Page=HomePage&NS=iControlREST>`_
+
+
+Connecting to BIG-IP
+````````````````````
+
+Any BIG-IP user with administrative rights can use the F5 Modules for Ansible.
+
+To secure the user's password so it is not stored in plain text in your playbook or inventory file, you can use `Ansible Vault <https://docs.ansible.com/ansible/2.4/vault.html>`_.
+
+You do not need to exchange key pairs between the machine running Ansible and BIG-IP, because the F5 modules use the API, rather than SSH, to connect.
+
+In addition, your playbook must contain ``delegate_to: localhost``, so that the modules run locally on the machine running Ansible. Without this setting, the modules try to run on BIG-IP and fail, because the supporting Python libraries are not on BIG-IP.
```

Comment:
The terminology here is a little off.

A Playbook, would contain `connection: local`.

Tasks of the playbook would contain `delegate_to: localhost`

The above are two ways of saying the same thing. The difference is that `connection: local` applies to all Tasks. Where-as `delegate_to: localhost` only applies to a single Task (and therefore must be re-specified for each Task you want to use it on).